src/OpenLoco/src/GameCommands/Buildings/RemoveBuilding.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "GameCommands/Buildings/CreateBuilding.h"
```

Comment:
This seems like an unnecessary include. Could it be removed?

Reply:
I see it was added due to the `BuildingPlacementArgs`-based constructor. Never mind then :)